recipes/simfil/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.62.0"
+
+
+class SimfilRecipe(ConanFile):
+    name = "simfil"
+    description = "simfil is a C++ 17 library and a language for querying structured map feature data. The library provides an efficient in-memory storage pool for map data, optimized for the simfil query language, along with a query interpreter to query the actual data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Klebert-Engineering/simfil"
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Always use SDPX short format: https://spdx.org/licenses/BSD-3-Clause.html